sporkfish/searcher/negamax_lazy_smp.py

```diff
@@ -47,19 +47,21 @@ def _start_search_from_root(
         :rtype: Tuple[float, chess.Move]
         """
 
-        # Let processes race down lazily and see who completes first
-        # We need to add more asymmetry but a task for later
         def task() -> Tuple[float, chess.Move]:
-            return self._start_search_from_root(board, depth, alpha, beta)
+            return NegamaxSp._start_search_from_root(self, board, depth, alpha, beta)
```

Comment:
It's just calling the parent class method and sending it to different processes - kinda sucks at the moment because transposition tables don't work very well and our move ordering is completely deterministic so there is close to 0 asymmetry